assets/section-footer.css

```diff
@@ -471,6 +493,23 @@ noscript .localization-selector.link {
   text-decoration: none;
 }
 
+
+
+.policies {
+  display: inline;
+}
+
+.policies li {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.policies li::before {
+  content: '\00B7';
+  padding: 0 0.8rem;
+}
+
```

Comment:
I think it looks a bit nicer at the bottom below the rest on a new row. With maybe a bit more white space in between the first and the second row. But when it's short without the policy links it's not bad under the payment icons. 🤔

Reply:
I think once we settle on a decision here I'm ok to approve 👍

Reply:
@ludoboludo @sofiamatulis @allylane We decided to move the policy links and copyright text to it's own line to avoid the issues [mentioned above](https://github.com/Shopify/dawn/pull/2084#discussion_r1019550762).

@allylane suggested above that we should center align this content, but I personally think that left aligned looks better in most scenarios.

### Left align
#### Payment Icons ON. Policy Links OFF.
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/37168033/206398922-05225add-4c83-44b4-9ab9-85210d557e3e.png">

#### Payment Icons ON. Policy Links ON.
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/37168033/206398557-ef3305e1-7730-4985-bc3a-bad6776338f6.png">

#### Payment Icons OFF. Policy Links OFF.
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/37168033/206399306-2e11bd22-8de2-433f-929a-e5facc09e39a.png">

#### Payment Icons OFF. Policy Links ON.
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/37168033/206399261-7f842f73-c241-4548-a1d1-5db312e85b08.png">

### Center aligned
#### Payment Icons ON. Policy Links OFF.
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/37168033/206399076-241e8170-fce2-4e14-a313-c20ca5aedefb.png">

#### Payment Icons ON. Policy Links ON.
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/37168033/206399197-8b60e776-3157-40a9-85ae-08feefe26b06.png">

#### Payment Icons OFF. Policy Links OFF.
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/37168033/206399367-af27d82d-611f-4dbc-b4b2-3cfb62dda150.png">

#### Payment Icons OFF. Policy Links ON.
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/37168033/206399487-16c4ee8c-3b84-4e51-bb2d-14b3da32b15c.png">

Do we want to move forward with `center` aligned, like originally suggested, or should we consider this `left` aligned?

Reply:
@ludoboludo @sofiamatulis @allylane We decided to move the policy links and copyright text to it's own line to avoid the issues [mentioned above](https://github.com/Shopify/dawn/pull/2084#discussion_r1019550762).

@allylane suggested above that we should center align this content, but I personally think that left aligned looks better in most scenarios.

### Left align
#### Payment Icons ON. Policy Links OFF.
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/37168033/206398922-05225add-4c83-44b4-9ab9-85210d557e3e.png">

#### Payment Icons ON. Policy Links ON.
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/37168033/206398557-ef3305e1-7730-4985-bc3a-bad6776338f6.png">

#### Payment Icons OFF. Policy Links OFF.
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/37168033/206399306-2e11bd22-8de2-433f-929a-e5facc09e39a.png">

#### Payment Icons OFF. Policy Links ON.
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/37168033/206399261-7f842f73-c241-4548-a1d1-5db312e85b08.png">

### Center aligned
#### Payment Icons ON. Policy Links OFF.
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/37168033/206399076-241e8170-fce2-4e14-a313-c20ca5aedefb.png">

#### Payment Icons ON. Policy Links ON.
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/37168033/206399197-8b60e776-3157-40a9-85ae-08feefe26b06.png">

#### Payment Icons OFF. Policy Links OFF.
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/37168033/206399367-af27d82d-611f-4dbc-b4b2-3cfb62dda150.png">

#### Payment Icons OFF. Policy Links ON.
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/37168033/206399487-16c4ee8c-3b84-4e51-bb2d-14b3da32b15c.png">

Do we want to move forward with `center` aligned, like originally suggested, or should we consider this `left` aligned?

Reply:
That's a good point. From an editing perspective, I do worry about introducing some styling magic with things moving around based on unrelated settings so that's why I suggested picking the one that works for most use cases (IMO, left). But from a code perspective, it shouldn't be that hard to make this dynamic.

@allylane Could you please verify if there are more scenarios we haven't covered that makes the policy links or copyright look off?

Are we okay with always left or do we want to introduce a setting-dependant dynamic solution?

Reply:
@ludoboludo I've updated to center-align whenever both language/currency settings are off. This should also account for when it's empty.